components/compliance-service/reporting/relaxting/nodes.go

```diff
@@ -174,13 +176,32 @@ func (backend *ES2Backend) GetNodes(from int32, size int32, filters map[string][
 		if err != nil {
 			return nil, emptyTotals, errors.Wrapf(err, "%s error retrieving node count totals: ", myName)
 		}
+		// sort nodes by the platform name and release
+		if sortField == "platform.name.lower" {
+			sortNodes(nodes, sortField, sortAsc)
+		}
 		return nodes, TotalNodeCounts{Total: int32(searchResult.TotalHits()), Passed: nodeSummary.Compliant, Failed: nodeSummary.Noncompliant, Skipped: nodeSummary.Skipped}, nil
 	}
 
 	logrus.Debugf("%s Found no nodes\n", myName)
 	return nodes, emptyTotals, nil
 }
 
+func sortNodes(nodes []*reportingapi.Node, sortField string, sortAsc bool) {
```

Comment:
would love to get a unit test for this function!

Reply:
sure, I will add the test case.

Reply:
actually, now that i read through pete's comments, probably no need for a test here, since as he pointed out this should be done in elasticsearch for pagination concerns